tests/phpunit/integration/Modules/Thank_With_Google/Web_TagTest.php

```diff
@@ -84,29 +96,52 @@ public function test_content_placeholder_inserted_static_below_content() {
 		$this->assertStringContainsString( '<button twg-button', $output );
 	}
 
+	public function test_content_counter_placeholder_not_inserted_static_below_content() {
+		$this->web_tag->set_cta_placement( Web_Tag::PLACEMENT_STATIC_BELOW_CONTENT );
+
+		$this->create_post_and_go_to_it();
+
+		$output = apply_filters( 'the_content', $this->dummy_content );
+
+		$this->assertStringStartsWith( $this->dummy_content, $output );
+		$this->assertStringContainsString( 'Thank with Google snippet added by Site Kit', $output );
+		$this->assertStringNotContainsString( '<div counter-button', $output ); }
+
 	public function test_content_placeholder_inserted_static_above_content() {
 		$this->web_tag->set_cta_placement( Web_Tag::PLACEMENT_STATIC_ABOVE_CONTENT );
 
 		$this->create_post_and_go_to_it();
 
 		$output = apply_filters( 'the_content', $this->dummy_content );
 
-		$this->assertStringEndsWith( $this->dummy_content, $output );
 		$this->assertStringContainsString( 'Thank with Google snippet added by Site Kit', $output );
 		$this->assertStringContainsString( '<button twg-button', $output );
+		$this->assertStringContainsString( '<div counter-button', $output );
 	}
 
 	public function test_content_placeholder_inserted_static_below_first_paragraph() {
-		$this->web_tag->set_cta_placement( Web_Tag::PLACEMENT_STATIC_AFTER_1ST_P );
+		$this->web_tag->set_cta_placement( Web_Tag::PLACEMENT_STATIC_BELOW_FIRST_P );
 
 		$this->create_post_and_go_to_it();
 
 		$output = apply_filters( 'the_content', $this->dummy_content );
 
-		$this->assertStringStartsWith( '<p>Hello World</p>', $output );
-		$this->assertStringEndsWith( '<p>Goodbye World</p>', $output );
+		$this->assertStringStartsWith(
+			'<p>Hello World</p>'
+			. PHP_EOL
+			. '<!-- Thank with Google snippet added by Site Kit -->'
+			. PHP_EOL
+			. '<button twg-button',
+			$output
+		);
+		$this->assertStringEndsWith(
```

Comment:
It would be nice to retain the check that `<p>Goodbye World</p>` still exists in the output, just for the sake of completion. How about adding it to this assertion?